examples/AccountManagement/GetAccountChanges.php

```diff
@@ -118,18 +119,13 @@ public static function runExample(GoogleAdsClient $googleAdsClient, $customerId)
         // the change status in each row.
         foreach ($response->iterateAllElements() as $googleAdsRow) {
             /** @var GoogleAdsRow $googleAdsRow */
-            // Note that the resource type and resource status printed below are enum values.
-            // For example, a value of 3 will be returned when the resource type is 'AD_GROUP'.
-            // A mapping of enum names to values can be found in the following files:
-            // * ChangeStatusResourceType.php (for resource type)
-            // * ChangeStatusOperation.php (for resource status)
             printf(
-                "On %s, change status '%s' shows resource '%s' with type %d and status %d.%s",
+                "On %s, change status '%s' shows resource '%s' with type '%s' and status '%s'.%s",
                 $googleAdsRow->getChangeStatus()->getLastChangeDateTime()->getValue(),
```

Comment:
Should we also change likes like this to `$googleAdsRow->getChangeStatus()->getLastChangeDateTimeValue()`?

@fiboknacky, WDYT?

Reply:
I thought Pierrick would do that as a part of other PRs.
But yes, please do it here as it's still somehow related changes and it saves number of PRs.

Reply:
Done